frontend/packages/knative-plugin/src/utils/create-knative-utils.ts

```diff
@@ -180,6 +180,8 @@ export const getKnativeServiceDepResource = (
     knativeServiceUpdated = _.omit(originalKnativeService, [
       'status',
       'spec.template.metadata.name',
+      'spec.template.spec.containers[0].livenessProbe',
+      'spec.template.spec.containers[0].readinessProbe',
```

Comment:
I think it is not the right solution for this issue. If you notice we create another container using the image from the internal image registry and health checks get added to the newly created container. So, here can we prevent to creating a new container in the edit flow? @jerolimov @Lucifergene 

In Create flow we do copy the external image to the internal registry and we use that to create a container but in the case of edit flow, we need to think about it.

One more thing that confuse the customer if we go with this fix. Before the edit there is no health check alert on the side panel of the KSVC but after the edit health alert start showing as there is no health check been omited from the first container. See the attached video


https://github.com/openshift/console/assets/2561818/b531d974-b6d7-4319-b74e-44c17290d6e6